File: php/Product.php
<?php
class Product
{
    // attribute Product
    private $idProduct = 0;
    private $price = 0;
    // constructor
    public function __construct($idProduct = 0, $price = 0)
    {
        $this->idProduct = $idProduct;
        $this->price = $price;
    }
    // getter setter idProduct
    public function getIdProduct()
    {
        return $this->idProduct;
    }
    public function setIdProduct($idProduct)
    {
        $this->idProduct = $idProduct;
    }
    // getter setter price
    public function getPrice()
    {
        return $this->price;
    }
    public function setPrice($price)
    {
        $this->price = $price;
    }
    // method untuk menampilkan semua attribute Product
    public function printProduct()
    {
        echo "ID Product: " . $this->getIdProduct() . "<br/>";
        // price di format agar tampil dalam format idr
        echo "Price: Rp." . number_format($this->getPrice(), 2, ",", ".") . "<br/>";
    }
}

File: python/Product.py
class Product:
    # contructroe
    def __init__(self):
        self.idProduct = 0
        self.price = 0
    # getter setter idProduct
    def getIdProduct(self):
        return self.idProduct
    
    def setIdProduct(self, idProduct):
        self.idProduct = idProduct
    # getter setter price
    def getPrice(self):
        return self.price
    
    def setPrice(self, price):
        self.price = price

    def printProduct(self):
        print("ID Product: {}".format(self.getIdProduct()))
        # format {.,} agar price tampil dalam format idr
        print("Price: Rp.{:,}".format(self.getPrice()))